test/fixtures/es-module-loaders/loader-this-value-inside-hook-functions.mjs

```diff
@@ -8,7 +8,7 @@ export function load(url, _, next) {
   return next(url);
 }
 
-export function globalPreload() {
+export function initialize() {
```

Comment:
Can we duplicate this test in `test/es-module/test-esm-loader-globalpreload-hook.mjs` for good measure?